pytorch_lightning/utilities/hydra.py

```diff
@@ -0,0 +1,71 @@
+import os
+from typing import List, Optional, Tuple
+
+import __main__
+import torch
+
+from pytorch_lightning.utilities import _HYDRA_AVAILABLE
+
+if _HYDRA_AVAILABLE:
+    from hydra.core.hydra_config import HydraConfig
+    from hydra.utils import get_original_cwd
+    from omegaconf import OmegaConf
+
+
+def get_ddp_spawn_command_for_hydra(command: List[str], local_rank: str) -> Tuple[str, str]:
```

Comment:
Local rank is an int

```suggestion
def get_ddp_spawn_command_for_hydra(command: List[str], local_rank: int) -> Tuple[str, str]:
```

Reply:
ah yes, it comes from the loop iterator.  for some reason thought it was coming from the environment variable